jf_agent/ratelimit.py

```diff
@@ -70,8 +70,7 @@ def limit(self, realm):
             sleep_period_secs = (sleep_until - datetime.utcnow()).total_seconds()
             if sleep_period_secs > 0:  # it's possible that sleep_until was a couple ms ago
                 logger.info(
-                    logging.INFO,
-                    f'Sleeping for {sleep_period_secs:.1f} secs ({sleep_period_secs / 60.0:.1f} mins)',
+                    f'Sleeping for {sleep_period_secs:.1f} secs ({sleep_period_secs / 60.0:.1f} mins)'
```

Comment:
This was the 'root' bug

`logger.info` DOES NOT accept a `logging.INFO` call. This logger.info call must have previously been `logger.log`, which WOULD take the `logging.INFO` argument as the logging level. When we changed this field from `logger.log` to `logger.info` we forgot to remove the `logging.INFO` arg